app/models/vote.rb

```diff
@@ -7,17 +7,50 @@ class Vote < ActiveRecord::Base
   has_many :vote_options, dependent: :destroy
   has_many :vote_posts, dependent: :destroy
 
-  validates :title, presence: true
+  validates :title, :status, presence: true
```

Comment:
> Where we going to require Agenda here?

We decided not to, so that it would be possible to create votes before the agendas were created